lms/static/js/spec/main.js

```diff
@@ -718,7 +718,9 @@
         'js/spec/edxnotes/collections/notes_spec.js',
         'js/spec/search/search_spec.js',
         'js/spec/navigation_spec.js',
-        'js/spec/courseware/updates_visibility.js',
+        'js/spec/courseware/updates_visibility_spec.js',
```

Comment:
Nit: I try to maintain these in alphabetical order (although I may be the only one!)